src/Components/ImagesTable/ImagesTable.js

```diff
@@ -215,111 +208,90 @@ const ImagesTable = () => {
               </ToolbarItem>
             </ToolbarContent>
           </Toolbar>
-          <Tabs
-                className="pf-u-ml-md"
-                activeKey={activeTabKey}
-                onSelect={handleTabClick}
-            >
-               
-          <Tab eventKey={0} title={<TabTitleText>EDGE</TabTitleText>}>
-            <AsyncComponent
-              appName="edge"
-              module="./Images"
-              // historyProp={useHistory}
-              navigateProp={useNavigate}
-              locationProp={useLocation}
-            /> 
-          </Tab>
-
-          <Tab eventKey={1} title={<TabTitleText>Image Builder</TabTitleText>}>
-            <TableComposable variant="compact" data-testid="images-table">
-              <Thead>
-                <Tr>
-                  <Th />
-                  <Th>Image name</Th>
-                  <Th>Created/Updated</Th>
-                  <Th>Release</Th>
-                  <Th>Target</Th>
-                  <Th>Status</Th>
-                  <Th>Instance</Th>
-                  <Th />
-                </Tr>
-              </Thead>
-              {composes.allIds
-                .slice(itemsStartInclusive, itemsEndExclusive)
-                .map((id, rowIndex) => {
-                  const compose = composes.byId[id];
-                  return (
-                    <Tbody key={id} isExpanded={isExpanded(compose)}>
-                      <Tr className="no-bottom-border">
-                        <Td
-                          expand={{
-                            rowIndex,
-                            isExpanded: isExpanded(compose),
-                            onToggle: () =>
-                              handleToggle(compose, !isExpanded(compose)),
-                          }}
```

Comment:
Maybe just drop the first 2 commits? Seems like they're negated here anyway.

Reply:
good advice, will do it